### PR TITLE
Include py38 to tox and travis config

### PR DESCRIPTION
**Pull request by mihrab34**

### CODE REVIEW

**Comment by luzfcb** [comment]
@mihrab34 Thanks to your contribution. Can you add yourself on the AUTHORS.rst ?

**Comment by mihrab34** [comment]
Thanks for the inclusion.

File: HISTORY.rst [hunk 1]
```diff
@@ -19,7 +19,8 @@ Important Changes:
 
 Other Changes:
 
-* Prevent ``click`` API v7.0 from showing choices when already shown thanks to `@rly`_ (#1168)
+* Test the codebase with python3.8 beta on tox and travis-ci, thanks to `@mihrab34`_ (#1168)
```
Comment (michaeljoseph) [inline HISTORY.rst]: This should be #1206 right?

Comment (luzfcb) [reply HISTORY.rst]: 😱 Good point

Comment (luzfcb) [reply HISTORY.rst]: Fixed on f8470a8c4b01e213074c8514d2710f2f990faa47

**Comment by hackebrot** [comment]
Thank you for your pull-request @mihrab34! 🍪

**Pull request merged**